<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VQJ3RG5" name="com.catchexceptions.seleniumpom.testcases.DataProviderExample" tests="3" failures="0" timestamp="13 Sep 2018 01:16:45 GMT" time="0.000" errors="0">
  <testcase name="verifyData1" time="0.000" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
  <testcase name="verifyData1" time="0.000" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
  <testcase name="verifyData1" time="0.000" classname="com.catchexceptions.seleniumpom.testcases.DataProviderExample"/>
</testsuite> <!-- com.catchexceptions.seleniumpom.testcases.DataProviderExample -->
